article/sections/_2_graph_importance.tex

```diff
@@ -94,18 +92,20 @@ \section{Perils of disregard}
    \label{fig:IND_GRAPH}
```

Comment:
In the presentation, you used these beautiful images where the arrows showing the perturbation were highlighted in red, according to each causal graph. Could you do that here? I believe you can set the color as an edge attribute for a given edge in graphviz. You should be able to do something like `graph.edge(A, B).update_attr(color="red")`